etc/gummi.profile

```diff
@@ -0,0 +1,13 @@
+# Firejail profile for latex-common
+# This file is overwritten after every install/update
+# Persistent local customizations
+include gummi.local
+# Persistent global definitions
+# added by caller profile
+#include globals.local
+
+private-bin gummi
```

Comment:
Works this? AFAIK gummi is a GUI-Software.

Reply:
Well you do have to call ```$ firejail gummi``` or set this in your .desktop file for Gummi.

Reply:
I mean, does gummi need to call /bin/latex or similar?

Reply:
Good thinking. Thank you. I explicitly disabled this behavior.
I'll get back to you with a fix.

Reply:
For your information:  
Based on https://github.com/alexandervdm/gummi/blob/8887f09fafe200c2602a94413d9350013fd97c6a/lib/latex_dvi.sh I will have to include ```latex```, ```dvipdf```, ```dvips``` and ```ps2pdf```.

Reply:
Dummy question: "Why does ```ignore include disable-interpreters.inc``` not work?"

Reply:
#2153, if you want a special interpreter to be available in the sandbox you can use `include allow-perl.inc` (or `allow-python3.inc`, ...)

Reply:
> #2153, if you want a special interpreter to be available in the sandbox you can use `include allow-perl.inc` (or `allow-python3.inc`, ...)

In my case (I include *latex-common.profile* which includes *disable-interpreters.inc*) python is not available even if I include *allow-python2.inc* and *allow-python3.inc*. (lua and perl have the same issue)
```
$ firejail --profile=/etc/firejail/gummi.profile --debug rubber
Reading profile /etc/firejail/gummi.profile
Reading profile /etc/firejail/latex-common.profile
Reading profile /etc/firejail/disable-common.inc
Reading profile /etc/firejail/disable-devel.inc
Reading profile /etc/firejail/disable-exec.inc
Reading profile /etc/firejail/disable-interpreters.inc
Reading profile /etc/firejail/disable-passwdmgr.inc
Reading profile /etc/firejail/disable-programs.inc
Reading profile /etc/firejail/allow-lua.inc
Reading profile /etc/firejail/allow-perl.inc
Reading profile /etc/firejail/allow-python3.inc
Reading profile /etc/firejail/allow-python2.inc
DISPLAY=:0 parsed as 0
Parent pid 101341, child pid 101368
Building quoted command line: 'rubber' 
Command name #rubber#
Building quoted command line: 'rubber' 
Command name #rubber#
Initializing child process
PID namespace installed
Mounting tmpfs on /run/firejail/mnt directory
Creating empty /run/firejail/mnt/seccomp directory
Creating empty /run/firejail/mnt/seccomp/seccomp.protocol file
Creating empty /run/firejail/mnt/seccomp/seccomp.postexec file
sbox run: /run/firejail/lib/fnet ifup lo 
Network namespace enabled, only loopback interface available
Build protocol filter: unix
sbox run: /run/firejail/lib/fseccomp protocol build unix /run/firejail/mnt/seccomp/seccomp.protocol 
Mounting /proc filesystem representing the PID namespace
Basic read-only filesystem:
Mounting read-only /etc
Mounting noexec /etc
Mounting read-only /var
Mounting noexec /var
Mounting read-only /bin
Mounting read-only /lib
Mounting read-only /usr
Mounting tmpfs on /var/lock
Mounting tmpfs on /var/tmp
Mounting tmpfs on /var/log
Create the new utmp file
Mount the new utmp file
Cleaning /home directory
Cleaning /run/user directory
Sanitizing /etc/passwd, UID_MIN 1000
Sanitizing /etc/group, GID_MIN 1000
Disable /run/firejail/network
Disable /run/firejail/bandwidth
Disable /run/firejail/name
Disable /run/firejail/profile
Disable /run/firejail/x11
Mounting tmpfs on /dev
Process /dev/shm directory
Copying files in the new bin directory
Checking /usr/local/bin/gummi
firejail exec symlink detected
Checking /usr/bin/gummi
sbox run: /run/firejail/lib/fcopy /usr/bin/gummi /run/firejail/mnt/bin 
Checking /usr/local/bin/latex
firejail exec symlink detected
Checking /usr/bin/latex
sbox run: /run/firejail/lib/fcopy /usr/bin/pdftex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/latex /run/firejail/mnt/bin 
Checking /usr/local/bin/pdftex
Checking /usr/bin/pdftex
sbox run: /run/firejail/lib/fcopy /usr/bin/pdftex /run/firejail/mnt/bin 
Checking /usr/local/bin/pdflatex
firejail exec symlink detected
Checking /usr/bin/pdflatex
sbox run: /run/firejail/lib/fcopy /usr/bin/pdftex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/pdflatex /run/firejail/mnt/bin 
Checking /usr/local/bin/xetex
Checking /usr/bin/xetex
sbox run: /run/firejail/lib/fcopy /usr/bin/xetex /run/firejail/mnt/bin 
Checking /usr/local/bin/xelatex
Checking /usr/bin/xelatex
sbox run: /run/firejail/lib/fcopy /usr/bin/xetex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/xelatex /run/firejail/mnt/bin 
Checking /usr/local/bin/luatex
Checking /usr/bin/luatex
sbox run: /run/firejail/lib/fcopy /usr/bin/luatex /run/firejail/mnt/bin 
Checking /usr/local/bin/lualatex
Checking /usr/bin/lualatex
sbox run: /run/firejail/lib/fcopy /usr/bin/luatex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/lualatex /run/firejail/mnt/bin 
Checking /usr/local/bin/rubber
Checking /usr/bin/rubber
sbox run: /run/firejail/lib/fcopy /usr/bin/rubber /run/firejail/mnt/bin 
Checking /usr/local/bin/latexmk
Checking /usr/bin/latexmk
file /usr/share/texmf-dist/scripts/latexmk/latexmk.pl not found
sbox run: /run/firejail/lib/fcopy /usr/bin/latexmk /run/firejail/mnt/bin 
Checking /usr/local/bin/perl
Checking /usr/bin/perl
sbox run: /run/firejail/lib/fcopy /usr/bin/perl /run/firejail/mnt/bin 
Checking /usr/local/bin/env
Checking /usr/bin/env
sbox run: /run/firejail/lib/fcopy /usr/bin/env /run/firejail/mnt/bin 
Checking /usr/local/bin/ps2pdf
Checking /usr/bin/ps2pdf
sbox run: /run/firejail/lib/fcopy /usr/bin/ps2pdf /run/firejail/mnt/bin 
Checking /usr/local/bin/dvipdf
Checking /usr/bin/dvipdf
sbox run: /run/firejail/lib/fcopy /usr/bin/dvipdf /run/firejail/mnt/bin 
Checking /usr/local/bin/dvips
Checking /usr/bin/dvips
sbox run: /run/firejail/lib/fcopy /usr/bin/dvips /run/firejail/mnt/bin 
Checking /usr/local/bin/synctex
Checking /usr/bin/synctex
sbox run: /run/firejail/lib/fcopy /usr/bin/synctex /run/firejail/mnt/bin 
Checking /usr/local/bin/tex
firejail exec symlink detected
Checking /usr/bin/tex
sbox run: /run/firejail/lib/fcopy /usr/bin/tex /run/firejail/mnt/bin 
file /usr/share/texmf-dist/scripts/context/stubs/unix/luatools not found
sbox run: /run/firejail/lib/fcopy /usr/bin/luatools /run/firejail/mnt/bin 
file /usr/share/texmf-dist/scripts/luaotfooad/luaotfooad-tool.lua not found
sbox run: /run/firejail/lib/fcopy /usr/bin/luaotfooad-tool /run/firejail/mnt/bin 
file /usr/share/texmf-dist/scripts/lua2dox/lua2dox_filter not found
sbox run: /run/firejail/lib/fcopy /usr/bin/lua2dox_filter /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/lua /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luatex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/lualatex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luac /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/lua5.2 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luac /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luac5.3 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luajittex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luatex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luac5.2 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luajit-2.0.5 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luajit /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/lua /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/lua5.3 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luajit-2.0.5 /run/firejail/mnt/bin 
file /usr/share/texmf-dist/scripts/context/stubs/unix/luatools not found
sbox run: /run/firejail/lib/fcopy /bin/luatools /run/firejail/mnt/bin 
file /usr/share/texmf-dist/scripts/luaotfooad/luaotfooad-tool.lua not found
sbox run: /run/firejail/lib/fcopy /bin/luaotfooad-tool /run/firejail/mnt/bin 
file /usr/share/texmf-dist/scripts/lua2dox/lua2dox_filter not found
sbox run: /run/firejail/lib/fcopy /bin/lua2dox_filter /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/lua /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luatex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/lualatex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/luac /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/lua5.2 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luac /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/luac5.3 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/luajittex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/luatex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/luac5.2 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luajit-2.0.5 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/luajit /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/lua /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/lua5.3 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/luajit-2.0.5 /run/firejail/mnt/bin 
file /usr/share/texmf-dist/scripts/context/stubs/unix/luatools not found
sbox run: /run/firejail/lib/fcopy /usr/sbin/luatools /run/firejail/mnt/bin 
file /usr/share/texmf-dist/scripts/luaotfooad/luaotfooad-tool.lua not found
sbox run: /run/firejail/lib/fcopy /usr/sbin/luaotfooad-tool /run/firejail/mnt/bin 
file /usr/share/texmf-dist/scripts/lua2dox/lua2dox_filter not found
sbox run: /run/firejail/lib/fcopy /usr/sbin/lua2dox_filter /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/lua /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luatex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/lualatex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/luac /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/lua5.2 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luac /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/luac5.3 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/luajittex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/luatex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/luac5.2 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luajit-2.0.5 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/luajit /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/lua /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/lua5.3 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/luajit-2.0.5 /run/firejail/mnt/bin 
file /usr/share/texmf-dist/scripts/context/stubs/unix/luatools not found
sbox run: /run/firejail/lib/fcopy /sbin/luatools /run/firejail/mnt/bin 
file /usr/share/texmf-dist/scripts/luaotfooad/luaotfooad-tool.lua not found
sbox run: /run/firejail/lib/fcopy /sbin/luaotfooad-tool /run/firejail/mnt/bin 
file /usr/share/texmf-dist/scripts/lua2dox/lua2dox_filter not found
sbox run: /run/firejail/lib/fcopy /sbin/lua2dox_filter /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/lua /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luatex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/lualatex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/luac /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/lua5.2 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luac /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/luac5.3 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/luajittex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/luatex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/luac5.2 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/luajit-2.0.5 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/luajit /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/lua /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/lua5.3 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/luajit-2.0.5 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/pythonz_install /run/firejail/mnt/bin 
file /usr/share/texmf-dist/scripts/pythontex/pythontex.py not found
sbox run: /run/firejail/lib/fcopy /usr/bin/pythontex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python2.7 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python2 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python2.7-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python2-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/pythonz /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python2.7 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python2.7-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/python3-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/python3.8 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/python3 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/python /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/pythonz_install /run/firejail/mnt/bin 
file /usr/share/texmf-dist/scripts/pythontex/pythontex.py not found
sbox run: /run/firejail/lib/fcopy /bin/pythontex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python2.7 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/python2 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python2.7-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/python2-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/python3.8-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/python-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/pythonz /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/python2.7 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /bin/python2.7-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/python3-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/python3.8 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/python3 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/python /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/pythonz_install /run/firejail/mnt/bin 
file /usr/share/texmf-dist/scripts/pythontex/pythontex.py not found
sbox run: /run/firejail/lib/fcopy /usr/sbin/pythontex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python2.7 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/python2 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python2.7-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/python2-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/python3.8-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/python-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/pythonz /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/python2.7 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/sbin/python2.7-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/python3-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/python3.8 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/python3 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/python /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/pythonz_install /run/firejail/mnt/bin 
file /usr/share/texmf-dist/scripts/pythontex/pythontex.py not found
sbox run: /run/firejail/lib/fcopy /sbin/pythontex /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python2.7 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/python2 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python2.7-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/python2-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/python3.8-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /usr/bin/python3.8-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/python-config /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/pythonz /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/python2.7 /run/firejail/mnt/bin 
sbox run: /run/firejail/lib/fcopy /sbin/python2.7-config /run/firejail/mnt/bin 
Mount-bind /run/firejail/mnt/bin on top of /usr/local/bin
Mount-bind /run/firejail/mnt/bin on top of /usr/bin
Moun169 programs installed in 152.97 ms
t-bind /run/firejail/mnt/bin on top of /bin
Mount-bind /run/firejail/mnt/bin on top of /usr/local/games
Mount-bind /run/firejail/mnt/bin on top of /usr/local/sbin
Mount-bind /run/firejail/mnt/bin on top of /usr/sbin
Mount-bind /run/firejail/mnt/bin on top of /sbin
Debug 423: new_name #/home/foo/.config/gummi#, whitelist
Debug 531: fname #/home/foo/.config/gummi#, cfg.homedir #/home/foo#
Debug 423: new_name #/home/foo/Documents/1fh#, whitelist
Debug 531: fname #/home/foo/Documents/1fh#, cfg.homedir #/home/foo#
Debug 423: new_name #/home/foo/1fh#, whitelist
Debug 531: fname #/home/foo/Documents/1fh#, cfg.homedir #/home/foo#
Debug 423: new_name #/home/foo/master_arbeit#, whitelist
Debug 531: fname #/home/foo/Documents/1fh/master_arbeit#, cfg.homedir #/home/foo#
Debug 423: new_name #/home/foo/ma_latex#, whitelist
Debug 531: fname #/home/foo/Documents/1fh/master_arbeit/ma_latex#, cfg.homedir #/home/foo#
Debug 423: new_name #/home/foo/ma_literature#, whitelist
Debug 531: fname #/home/foo/Documents/1fh/master_arbeit/ma_literature#, cfg.homedir #/home/foo#
Debug 423: new_name #/home/foo/ma_illustrations#, whitelist
Debug 531: fname #/home/foo/Documents/1fh/master_arbeit/ma_illustrations#, cfg.homedir #/home/foo#
Debug 423: new_name #/home/foo/sem2#, whitelist
Debug 531: fname #/home/foo/Documents/1fh/sem2#, cfg.homedir #/home/foo#
Debug 423: new_name #/home/foo/sem3#, whitelist
Debug 531: fname #/home/foo/Documents/1fh/sem3#, cfg.homedir #/home/foo#
Debug 423: new_name #/home/foo/texmf#, whitelist
Debug 531: fname #/home/foo/texmf#, cfg.homedir #/home/foo#
Debug 423: new_name #/tmp/.X11-unix#, whitelist
Warning: cleaning all supplementary groups
Mounting tmpfs on /home/foo/.cache
2541 2507 0:158 / /home/foo/.cache rw,nosuid,nodev,relatime - tmpfs tmpfs rw,mode=700,uid=1000,gid=1000
mountid=2541 fsname=/ dir=/home/foo/.cache fstype=tmpfs
Generate private-tmp whitelist commands
blacklist /run/user/1000/bus
blacklist /run/dbus/system_bus_socket
Mounting read-only /proc/sys
Remounting /sys directory
Disable /sys/firmware
Disable /sys/hypervisor
Disable /sys/power
Disable /sys/kernel/debug
Disable /proc/sys/fs/binfmt_misc
Disable /proc/sys/kernel/core_pattern
Disable /proc/sys/kernel/modprobe
Disable /proc/sysrq-trigger
Disable /proc/sys/vm/panic_on_oom
Disable /proc/irq
Disable /proc/bus
Disable /proc/sched_debug
Disable /proc/timer_list
Disable /proc/kallsyms
Disable /usr/lib/modules (requested /lib/modules)
Disable /boot
Disable /run/user/1000/gnupg
Disable /run/user/1000/systemd
Disable /proc/kmsg
Replaced whitelist path: whitelist /home/foo/.config/gummi
.
.
.
Disable /usr/bin/lua (requested /usr/local/bin/lua5.3)
Disable /usr/bin/luajit-2.0.5 (requested /usr/local/bin/luajit)
Disable /usr/local/bin/luajit-2.0.5
Disable /usr/local/bin/luac5.2
Disable /usr/local/bin/luajittex
Disable /usr/bin/luac (requested /usr/local/bin/luac5.3)
Disable /usr/local/bin/lua5.2
Disable /usr/local/bin/luac
Disable /usr/local/bin/lua
Disable /usr/share/texmf-dist/scripts/lua2dox/lua2dox_filter (requested /usr/local/bin/lua2dox_filter)
Disable /usr/share/texmf-dist/scripts/luaotfooad/luaotfooad-tool.lua (requested /usr/local/bin/luaotfooad-tool)
Disable /usr/share/texmf-dist/scripts/context/stubs/unix/luatools (requested /usr/local/bin/luatools)
Disable /usr/bin/luatex (requested /usr/local/bin/lualatex)
Disable /usr/local/bin/luatex
Disable /usr/bin/lua (requested /usr/bin/lua5.3)
Disable /usr/bin/luajit-2.0.5 (requested /usr/bin/luajit)
Disable /usr/bin/luajit-2.0.5
Disable /usr/bin/luac5.2
Disable /usr/bin/luajittex
Disable /usr/bin/luac (requested /usr/bin/luac5.3)
Disable /usr/bin/lua5.2
Disable /usr/bin/luac
Disable /usr/bin/lua
Disable /usr/share/texmf-dist/scripts/lua2dox/lua2dox_filter (requested /usr/bin/lua2dox_filter)
Disable /usr/share/texmf-dist/scripts/luaotfooad/luaotfooad-tool.lua (requested /usr/bin/luaotfooad-tool)
Disable /usr/share/texmf-dist/scripts/context/stubs/unix/luatools (requested /usr/bin/luatools)
Disable /usr/bin/luatex (requested /usr/bin/lualatex)
Disable /usr/bin/luatex
Disable /usr/lib/lua
Disable /usr/share/lua
Disable /usr/local/bin/perl
Disable /usr/bin/perl
Disable /usr/lib/perl5
Disable /usr/share/perl5
Disable /usr/share/perl-image-exiftool
Disable /usr/lib/ruby
Disable /usr/bin/python2.7-config (requested /usr/local/bin/python2-config)
Disable /usr/local/bin/python2.7-config
Disable /usr/bin/python2.7 (requested /usr/local/bin/python2)
Disable /usr/local/bin/python2.7
Disable /usr/bin/python2.7-config (requested /usr/bin/python2-config)
Disable /usr/bin/python2.7-config
Disable /usr/bin/python2.7 (requested /usr/bin/python2)
Disable /usr/bin/python2.7
Disable /usr/lib/python2.7
Disable /usr/bin/python3.8 (requested /usr/local/bin/python3)
Disable /usr/local/bin/python3.8
Disable /usr/bin/python3.8-config (requested /usr/local/bin/python3-config)
DBlacklist violations are logged to syslog
isable /usr/local/bin/python3.8-config
Disable /usr/bin/python3.8 (requested /usr/bin/python3)
Disable /usr/bin/python3.8
Disable /usr/bin/python3.8-config (requested /usr/bin/python3-config)
Disable /usr/bin/python3.8-config
Disable /usr/lib/python3.8
Disable /usr/lib/python3.6
Disable /usr/lib/python3.8 (requested /usr/lib64/python3.8)
Disable /usr/lib/python3.6 (requested /usr/lib64/python3.6)
Mounting read-only /tmp/.X11-unix
2687 2629 0:46 /.X11-unix /tmp/.X11-unix ro,nosuid,nodev,noexec master:65 - tmpfs tmpfs rw
mountid=2687 fsname=/.X11-unix dir=/tmp/.X11-unix fstype=tmpfs
Disable /sys/fs
Disable /sys/module
disable pulseaudio
blacklist /run/user/1000/pulse/native
blacklist /run/user/1000/pulse/native
Create the new ld.so.preload file
DISPLAY=:0 parsed as 0
 line  OP JT JF    K
=================================
 0000: 20 00 00 00000004   ld  data.architecture
 0001: 15 01 00 c000003e   jeq ARCH_64 0003 (false 0002)
 0002: 06 00 00 7fff0000   ret ALLOW
 0003: 20 00 00 00000000   ld  data.syscall-number
 0004: 15 01 00 00000029   jeq socket 0006 (false 0005)
 0005: 06 00 00 7fff0000   ret ALLOW
 0006: 20 00 00 00000010   ld  data.args[0]
 0007: 15 00 01 00000001   jeq 1 0008 (false 0009)
 0008: 06 00 00 7fff0000   ret ALLOW
 0009: 06 00 00 0005005f   ret ERRNO(95)
Mount the new ld.so.preload file
Current directory: /home/foo
Install protocol filter: unix
configuring 10 seccomp entries in /run/firejail/mnt/seccomp/seccomp.protocol
sbox run: /usr/lib/firejail/fsec-print /run/firejail/mnt/seccomp/seccomp.protocol 
Mounting read-only /run/firejail/mnt/seccomp
Dropping all capabilities
noroot user namespace installed
Dropping all capabilities
NO_NEW_PRIVS set
Drop privileges: pid 1, uid 1000, gid 1000, nogroups 1
No supplementary groups
]0;firejail rubber Child process initialized in 226.92 ms
starting application
LD_PRELOAD=(null)
execvp argument 0: rubber
Searching $PATH for rubber
trying #/usr/share/fish /home/foo/.local/bin//rubber#
trying #/usr/share/fish/rubber#
trying #/home/foo/.local/bin//rubber#
trying #/usr/local/bin/rubber#
Installing /run/firejail/mnt/seccomp/seccomp.protocol seccomp filter

Parent is shutting down, bye...
```

Reply:
Where do you include allow-python3.inc? It must come _before_ disable-interpreters.inc.

Reply:
Does that help or do you need more info?

Reply:
> Where do you include allow-python3.inc? It must come _before_ disable-interpreters.inc.

I thought it would be something like this. I used my old config for testing, I had *latex-common.profile* included at the top.
Really appreciate your help.